pkg/helpers/helpers.go

```diff
@@ -49,6 +50,19 @@ func JSONMarshal(content interface{}, escape bool) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+// CloneAndConform doesn't change the content, it makes copy of content, and conform fields
+func CloneAndConform(src, dst interface{}) error {
```

Comment:
where is this new function being used?

Reply:
It will mark secret value as 'redacted'. This function will used by RP.

Reply:
Why not put the function in the RP codebase then? Since it won't be called anywhere in acs-engine code

Reply:
@zqingqing1 will update the PR with this new annotation to aks api model. We need this function here to be able to unit test it.

Reply:
Sounds good. Let's add a comment to explain that this function is needed for RP so that someone doesn't go in and remove it when they see it's not called.